### DISCUSSION TITLE
unable to set feed rate burn black with low PWR

### DISCUSSION THREAD

**Discussion by caracal22** [arkypita/LaserGRBL#1706] [Q&A]
feed rate

**Comment by StuartB4** [comment]
Engraving speed. 

![Untitled](https://user-images.githubusercontent.com/16980026/148294976-acc1a414-159d-436e-994f-2df1525fd2f5.png)


Or Border and Filling speed if you are doing a vector. 

![Untit2led](https://user-images.githubusercontent.com/16980026/148295353-301cfb9b-ad88-4671-87aa-95e769cc3584.png)
